sharepoint/sharepoint-ps/sharepoint-online/Export-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,64 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+---
+
+# Export-SPODataAccessGovernanceInsight
+
+## SYNOPSIS
+
+This commands exports/downloads the DAG report to the default path "C:\WINDOWS\system32\"
```

Comment:
>DAG [](http://example.com/codeflow?start=36&length=3)

first instance of mentioning an acronym: spell it out, put the acronym after in brackets